test/test_lt.py

```diff
@@ -12,3 +14,38 @@ def test_lt_matrix(self):
         A = base.lt([a+b, 2*a-b])
         assert str(A) == 'Lt(a) = a + b\nLt(b) = 2*a - b'
         assert str(A.matrix()) == 'Matrix([[1, 2], [1, -1]])'
+
+
+class TestMlt(unittest.TestCase):
+
+    def test_basic(self):
+        # from TensorDef.py
+        coords = symbols('t x y z', real=True)
+        st4d, g0, g1, g2, g3 = Ga.build('gamma*t|x|y|z', g=[1, -1, -1, -1],
+                                        coords=coords)
+
+        A = st4d.mv('T', 'bivector')
+
+        def TA(a1, a2):
+            return A | (a1 ^ a2)
+
+        T = Mlt(TA, st4d)
+
+        # tests begin
+
+        a1 = st4d.mv('a1', 'vector')
+        a2 = st4d.mv('a2', 'vector')
+        a3 = st4d.mv('a3', 'vector')
+        a4 = st4d.mv('a4', 'vector')
+
+        # calling the Mlt is like calling the function
+        assert T(a1, a2) == TA(a1, a2)
+
+        # for addition, argument slots are reused
+        assert (T + T)(a1, a2) == T(a1, a2) + T(a1, a2)
+        assert (T - T)(a1, a2) == T(a1, a2) - T(a1, a2)
+
+        # for multiplication, argument slots are chained
+        assert (T * T)(a1, a2, a3, a4) == TA(a1, a2) * T(a3, a4)
+        assert (T ^ T)(a1, a2, a3, a4) == TA(a1, a2) ^ T(a3, a4)
+        assert (T | T)(a1, a2, a3, a4) == TA(a1, a2) | T(a3, a4)
```

Comment:
More cases can be added:

```python
        alpha = st4d.mv('alpha', 'scalar')
        b  = st4d.mv('b', 'vector')

        # test other multilinear function properties
        assert T(alpha * a1, a2) == alpha * T(a1, a2)
        assert T(a1, alpha * a2) == alpha * T(a1, a2)
        assert T(a1 + b, a2) == T(a1, a2) + T(b, a2)
        assert T(a1, a2 + b) == T(a1, a2) + T(a1, b)
```

Tested locally to be working.

Reply:
I'm not sure that test is interesting - we already proved that `T(x) == TA(x)`, so your test is just showing that `TA` is multilinear, which isn't a test of `Mlt` at all.

Reply:
Tests don't need to be interesting. They simply serve as verifying its dumb correctness.

These properties are specified in the docstring,  so it's the proper place to add them.

Reply:
I'll add them in a follow-up, since then we can see if they affect coverage too (and CI is already done here anyway)